docs/user/security/authentication/index.asciidoc

```diff
@@ -42,6 +49,11 @@ xpack.security.authc.providers:
     enabled: false
 --------------------------------------------------------------------------------
 
+[role="screenshot"]
+image::user/security/images/kibana-login.png["Login Selector UI"]
```

Comment:
nit: given the configuration above, shouldn't I expect to see the Kerberos realm in this screenshot?

Reply:
Nope it's `enabled: false` :slightly_smiling_face:

Reply:
I kept it to show that `enabled` also affects selector (obviously though), but I can remove if it's more confusing than helpful?

Reply:
Ah I missed that!